tests/containers/rootless_podman.pm

```diff
@@ -60,7 +58,7 @@ sub run {
     }
     clean_container_host(runtime => $runtime);
     $self->select_serial_terminal();
-    assert_script_run "chmod -R 600 /etc/zypp/credentials.d/*" if is_sle;
+    assert_script_run "setfacl -x u:$user /etc/zypp/credentials.d/*" if is_sle;
```

Comment:
IMO this should be part of post_fail_hook and post_run_hook.

Reply:
just be careful with the $user

Reply:
> IMO this should be part of post_fail_hook and post_run_hook.

you mean that we should **revert** this in post_* hooks , right ? yeah make sense .

Reply:
Unfortunately, there is no guarantee that the code will reach this point

Reply:
Done